# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/mdadm/0001-mdadm-musl-remove-deprecated-values.h.patch
# Copyright (C) 2012 - 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/super-ddf.c	2012-11-25 01:35:58.911682709 +0100
+++ b/super-ddf.c	2012-11-24 23:24:00.900425167 +0100
@@ -29,7 +29,6 @@
 #include "mdadm.h"
 #include "mdmon.h"
 #include "sha1.h"
-#include <values.h>
 
 /* a non-official T10 name for creation GUIDs */
 static char T10[] = "Linux-MD";
@@ -971,13 +970,13 @@
 	{ DDF_RAID5E, LEVEL_UNSUPPORTED },
 	{ DDF_RAID5EE, LEVEL_UNSUPPORTED },
 	{ DDF_RAID6, 6},
-	{ MAXINT, MAXINT }
+	{ INT_MAX, INT_MAX }
 };
 
 static int map_num1(struct num_mapping *map, int num)
 {
 	int i;
-	for (i=0 ; map[i].num1 != MAXINT; i++)
+	for (i=0 ; map[i].num1 != INT_MAX; i++)
 		if (map[i].num1 == num)
 			break;
 	return map[i].num2;
@@ -2641,10 +2640,10 @@
 	if (!dev) {
 		/* Initial sanity check.  Exclude illegal levels. */
 		int i;
-		for (i=0; ddf_level_num[i].num1 != MAXINT; i++)
+		for (i=0; ddf_level_num[i].num1 != INT_MAX; i++)
 			if (ddf_level_num[i].num2 == level)
 				break;
-		if (ddf_level_num[i].num1 == MAXINT) {
+		if (ddf_level_num[i].num1 == INT_MAX) {
 			if (verbose)
 				fprintf(stderr, Name ": DDF does not support level %d arrays\n",
 					level);
--- a/super-intel.c	2012-11-24 23:26:46.412202198 +0100
+++ b/super-intel.c	2012-11-24 23:27:25.229092400 +0100
@@ -22,7 +22,6 @@
 #include "mdmon.h"
 #include "sha1.h"
 #include "platform-intel.h"
-#include <values.h>
 #include <scsi/sg.h>
 #include <ctype.h>
 #include <dirent.h>
